grass7/imagery/i.sentinel/i.sentinel.import/i.sentinel.import.py

```diff
@@ -66,6 +66,11 @@
 #% description: Name for output file to use with t.register
 #% required: no
 #%end
+#%option G_OPT_M_DIR
+#% key: metadata
+#% description: Name of directory into which Sentinel metadata json dumps are saved
```

Comment:
How about predefining a directory? I'd like that, it could go into LOC/MAP/json/ or so and being used by  i.sentinel.mask.py as well (hence, less "thinking" for the user where to store this file which should really be next to the data)

Reply:
@neteler should the file then always be written, or should another flag be added?

Reply:
I added a flag -f for this

Reply:
In order to maintain accordance with the current GRASS database structure, such a file should go to LOCATION_NAME/MAPSET/cell_misc/<raster map name>/ for individual rasters or to LOCATION_NAME/MAPSET/group/<group name>/ for imagery groups.